project/Dependencies.scala

```diff
@@ -104,6 +105,7 @@ object Dependencies {
     jacksonJdk8,
     jacksonJsr310,
     jacksonParameterNames,
+    webflux,
```

Comment:
I realise now that we introduced a spring dependency to the java-sdk. We should find another way to solve that. 

We need to convert the exception at the level of the `KalixClientImpl` instead.

Reply:
fixed [here](https://github.com/lightbend/kalix-jvm-sdk/pull/1473) along with a few bits remaining from your review